service.go

```diff
@@ -210,11 +212,15 @@ func New(name, version string) *Builder {
 	if version == "" {
 		version = "dev"
 	}
+	if env == "" {
+		env = "unspecified"
```

Comment:
we should allow empty for those who do not want this feature. Adding more data in the logging system without any need should probably be avoided.